### ISSUE TITLE
favicon in index.html

### ISSUE THREAD

**Issue by podotki** [ReactJSResources/react-webpack-babel#322]
Hi,

I can't find a way to display my favicon which I put in the same place than my index.html so this `<link rel="shortcut icon" href="favicon.ico">` is not working.

Do you have an idea ?

Thanks a lot for your help :)

**Comment by pgerochi** [comment]
hey @podotki, we don't have an index.html. If you want to put something a favicon, you can put it on here in the head tag: react-webpack-babel > src/> template.html - hope that helps!

**Comment by krustypop** [comment]
Oh sorry i forgot to say that I have renamed the template.htmt, is that a problem ?

**Comment by pgerochi** [comment]
@krustypop no, i think that's fine. You just need to change **webpack.config.js** and **webpack.production.config.js** from 

> template: './src/template.html', to template: './src/index.html',

**Comment by pgerochi** [comment]
@krustypop @podotki - did you manage to fix this? can we close it?
